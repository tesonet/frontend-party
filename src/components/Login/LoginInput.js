import styled from 'styled-components';

export default styled.input`
  width: 305px;
  height: 58px;
  border: none;
  border-radius: 5px;
  color: #999;
  padding-left: 55px;
  font-size: 16px;
  font-weight: 300;
  letter-spacing: 0.4px;
  &&:focus {
    outline: none;
  }
  &&::placeholder {
    font-size: 16px;
    color: #b3b3b3;
    letter-spacing: 0.4px;
    font-weight: 300;
  }
  @media (max-width: 400px) {
    width: calc(90% - 55px);
    margin: 0 auto;
  }
`;
