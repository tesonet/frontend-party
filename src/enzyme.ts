import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import fetchMock from "jest-fetch-mock";

declare let global: { fetch: {} };

global.fetch = fetchMock;
configure({ adapter: new Adapter() });
