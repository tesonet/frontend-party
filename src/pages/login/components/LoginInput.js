import React from "react";

export const LoginInput = ({
	handleChange,
	value,
	name,
	type,
	placeholder
}) => (
	<div className={`input-container-${name}`}>
		<input
			onChange={handleChange}
			value={value}
			name={name}
			type={type}
			placeholder={placeholder}
		/>
	</div>
);