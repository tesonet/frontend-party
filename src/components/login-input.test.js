import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";

import LoginInput from "./login-input";


const component = shallow(<LoginInput/>);

it('renders correctly', () => {
    expect(toJson(component)).toMatchSnapshot();
});
