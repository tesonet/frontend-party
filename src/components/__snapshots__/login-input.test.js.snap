// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <styled.label />
  <styled.input
    icon={[Function]}
    onChange={[Function]}
    required={true}
    type="text"
  />
</Fragment>
`;
